### ISSUE TITLE
Workflow: Devbox Run

### ISSUE THREAD

**Issue by Lagoja** [jetpack-io/devbox#94]
## Current State

If you want to run a one off command or package with Devbox, you currently need to start your shell, run your package, and then exit the shell. This adds a lot of steps and friction for developers who may want to quickly start a package or run a project in their shell

## Desired State

I set up a shell in advance with `devbox init`, and then add packages using`devbox add`. I now want to run a specific command or script in my Devbox Shell and then exit back to my normal shell, so I invoke my script or command with the Devbox CLI. This would be used to run my script in an isolated shell with all it’s dependencies, display the results, and then exit the Devbox shell.

**Comment by mentos1386** [comment]
This is something i see missing as well. I would like to use devbox inside CI and pre-commit hooks.

**Comment by Lagoja** [comment]
Hey everyone, this is now available in Devbox 0.0.7 -- you can run a one off command in your devbox shell using: 

```bash
devbox shell -- <cmd>
```

This will launch a shell based on your devbox.json, run your command, and then exit back to your normal shell. For example:

```bash
~ $  devbox shell -- echo "Hello World"
Installing nix packages. This may take a while...
Starting a devbox shell...
Hello World
~ $ 
```

More info is in our new CLI Docs: https://www.jetpack.io/devbox/docs/cli_reference/devbox_shell/
